python/py-validictory/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-validictory
+revision            0
+version             1.1.2
+
+categories-append   devel
+license             BSD
```

Comment:
PyPI isn't completely clear about this, but the [`setup.py`](https://github.com/jamesturk/validictory/blob/6571d051014a3fc302ecd8d67191cd07a6973fdd/setup.py#L16) states it's MIT

Reply:
@reneeotten yes, you're right. I guess copy and paste typo.